### ISSUE TITLE
WindowsAzure.Storage 

### ISSUE THREAD

**Issue by StevenLangeraert** [CrumpledDog/Umbraco-AzureLogger#11]
Showing the log detail fails silently when using latest version (8.1.1) of the WindowsAzure.Storage nuget package,
due to removed signature Microsoft.WindowsAzure.Storage.Table.(Retrieve(String, String, List<String>))

https://msdn.microsoft.com/library/azure/microsoft.windowsazure.storage.table.tableoperation.retrieve.aspx

**Comment by Jeavon** [comment]
@StevenLangeraert could you please the latest pre-release as I'm pretty sure we fixed this and upgraded the Storage dependency?

**Comment by StevenLangeraert** [comment]
@Jeavon Thanks for the fast reply. Could it be there was a major breaking WindowsAzure.Storage nuget upgrade since then? Since I see comments on upgrading to WindowsAzure.Storage 7.2.1 but not to 8.0.0, neither can I find an implementation in source code that matches the Retrieve signatures of the latest (8.1.1) WindowsAzure.Storage nuget

**Comment by Jeavon** [comment]
It's possible although it seems the breaking change in WindowsAzure.Storage are undocumented,  did you try the pre-release?
I'm happy to upgrade to v8.0.0 if needed...

**Comment by StevenLangeraert** [comment]
With pre-release you mean 

Install-Package Our.Umbraco.AzureLogger -Pre ?

or

Install-Package Our.Umbraco.AzureLogger -Version 0.2.1-alpha-000174

**Comment by Jeavon** [comment]
0.2.1-alpha-000174 from MyGet

**Comment by StevenLangeraert** [comment]
Yes! That version is compatible with 8.1.1. not that I realy need that version, but I just like to use the latest ;-).

Sorry I overlooked that version on MyGet. Feel free to close the ticket ;-)

Thx for the cool plugin!

**Comment by Jeavon** [comment]
Great, think we will make a release of this today/tomorrow
